qemu/tests/usb_redir.py

```diff
@@ -258,6 +298,8 @@ def _usb_stick_io(mount_point):
         return
 
     if not _usb_dev_verify():
+        if backend == 'tcp_socket':
+            process.getoutput("killall usbredirserver")
```

Comment:
It is just a clean step, suggest  `process.system(***, ignore_status=True)`

Reply:
Updated.